src/util.cpp

```diff
@@ -254,13 +254,6 @@ FileExistsFunction SetFileExistsFunction(
   return previous_function;
 }
 
-bool SaveFile(const char* name, const char* buf, size_t len, bool binary) {
```

Comment:
This is a general utility function, and a great example of something someone could be using even if not in this repo, and this deletion would break them.